docs/api-v1.md

```diff
@@ -288,19 +292,36 @@ GET /v1/events/
 
 Parameters:
 
-| Field     | Description                          | Required |
-| --------- | ------------------------------------ | -------- |
-| group     | To get all events by the group       | ❌        |
-| eventType | To get all events by type            | ❌        |
-| limit     | Limit the amount of events retrieved | ❌        |
-| offset    | Skip the fist _x_ events             | ❌        |
+| Field      | Description                          | Required |
+| ---------  | ------------------------------------ | -------- |
+| groupId    | To get all events by the group       | ❌        |
```

Comment:
I would revert the renaming and not include the id. I think it is unecessary to expicitly have `id` in the parameter name. Furthermore we did not do this for all other endpoints. So if we change it to `xxxId` we would have to adopt all other endpoints too.

```suggestion
| group      | To get all events by the group id    | ❌        |
```

Reply:
This is possible, but we need to define a default limit/page size. Default offset will be 0. What do you think about this?